arcade/hitbox/base.py

```diff
@@ -138,18 +240,28 @@ def __init__(
         scale: Tuple[float, float] = (1.0, 1.0),
     ):
         super().__init__(points, position=position, scale=scale)
-        self._angle = angle
+        self._angle: float = angle
 
     @property
-    def angle(self):
+    def angle(self) -> float:
+        """
+        The angle to rotate the raw points by in degrees
+        """
         return self._angle
 
     @angle.setter
     def angle(self, angle: float):
         self._angle = angle
         self._adjusted_cache_dirty = True
 
-    def get_adjusted_points(self):
+    def get_adjusted_points(self) -> PointList:
+        """
+        Return the offset, scaled, & rotated points of this hitbox.
+
+        As with :py:meth:`.HitBox.get_adjusted_points`, this method only
+        recalculates the adjusted values when necessary.
+        :return:
+        """
```

Comment:
Do we want to avoid the duplicated docstring here, instead writing a description on `HitBox` that can apply generically to all subclasses?

Reply:
I think it may be worth it to be explicit about the rotation for now. To my understanding, doing otherwise requires:

1. Getting the inheritance of docstrings and methods working first, which may be hard
2. Trusting the user to understand the idea of transformations on points

Please correct me if I'm wrong, because I'd love to be wrong about item 1.

The second concern is less important than I initially thought, as I found out these classes were intended by einarf and Clepto to be for advanced users only. This may be why they aren't included in the API doc at the moment. I'd be tempted to agree if we had a better sprite / texture generation API set up beyond `SpriteCircle` and `SpriteSolidColor`, but it's not a core 3.0 goal so I want to leave some notes in place for both ambitious beginners and my future 4am weekend self.

Reply:
I'm not sure.  "Tranformation" is definitely the correct term for what we're doing, where different `HitBox` implementations support different transformations.  I have no strong preference here.

Reply:
I'll leave this as is for now & mark it as resolved.